simulation/interfaces/interface_utils.py

```diff
@@ -88,7 +88,7 @@ def get_train_dataset_params(input_params: dict, old_params: Optional[dict] = No
     train_dataset_params['cache_limit'] = input_params['cache_limit']
     train_dataset_params['shuffle'] = input_params['shuffle']
     train_dataset_params['shuffle_algo'] = input_params['shuffle_algo']
-    train_dataset_params['shuffle_block_size'] = number_abbrev_to_int(
+    train_dataset_params['shuffle_block_size'] = normalize_count(
```

Comment:
have you been able to test these changes? could you make sure these numbers are displaying and being passed as intended by running `simulator` and passing in values for these parameters?

Reply:
If you could add a short "testing" section to the PR description that would be great as well!

Reply:
what is the procedure for checking i didn't break the simulator?